lib/src/parsers/parsers.dart

```diff
@@ -0,0 +1,30 @@
+import 'dart:ui';
+
+import 'package:vector_tile_renderer/src/expressions/expression.dart';
+
+import 'boolean.dart';
+import 'color.dart';
+import 'double.dart';
+import 'parser.dart';
+
+final Map<Type, Parser> _parsers = {
+  double: DoubleParser(),
+  bool: BooleanParser(),
+  Color: ColorParser(),
+};
+
+/// The common parser has no logic for parsing any special cases and can serve
+///  as a parser for everything not specified in _parsers.
+class CommonParser<T> extends Parser<T> {
```

Comment:
I did have it named NoOpParser at first but that's not entirely accurate since it still calls `_parseCommonExpressions` internally so I moved away from that. This is useful for parsing expressions that do not need a specialized parser like String which can still make use from the common expressions. Maybe `DefaultParser` is a fitting name?